config/sync/config_ignore.settings.yml

```diff
@@ -0,0 +1,9 @@
+ignored_config_entities:
+  - 'core.extension:theme'
+  - 'system.theme:default'
+  - 'block.block.*'
+  - '~block.block.seven_*'
+  - '~block.block.stanford_basic_*'
```

Comment:
We will need to remember to add the soe_subtheme to the soe_profile when we move this over.